fmriprep/workflows/bold/resampling.py

```diff
@@ -164,7 +164,7 @@ def select_target(subject_id, space):
     return workflow
 
 
-def init_bold_mni_trans_wf(template, mem_gb, omp_nthreads,
+def init_bold_mni_trans_wf(template, freesurfer, mem_gb, omp_nthreads,
```

Comment:
You're going to need to add a `freesurfer` parameter to any call of `init_bold_mni_trans_wf`, both in the code and in the documentation.

## Documentation

1. The docstring below:

https://github.com/poldracklab/fmriprep/blob/3b41b17e4e527b9bcf36d8293acab7b6a3a5e74a/fmriprep/workflows/bold/resampling.py#L181-L184

2. workflows.rst:

https://github.com/poldracklab/fmriprep/blob/3b41b17e4e527b9bcf36d8293acab7b6a3a5e74a/docs/workflows.rst#L454-L458

I would do `freesurfer=True` for these.

## Code

https://github.com/poldracklab/fmriprep/blob/3b41b17e4e527b9bcf36d8293acab7b6a3a5e74a/fmriprep/workflows/bold/confounds.py#L532-L541

I would put `freesurfer=False`, here.

Reply:
Ah. Sounds like you have some local changes. First, I would decide if you want to keep them:

```Shell
git diff
```

If you don't care to keep them, just run

```Shell
git checkout .
```

If you do, then you need to decide whether you want them in this pull request. If so:

```Shell
git stash
git pull
git stash apply
```

This might require resolving conflicts, which I can't really prescribe for you. Here's a very basic [how-to](https://githowto.com/resolving_conflicts), but there may be better ones you can google for.

If you want to keep the changes, but not as part of this pull request, I would put them on a new branch, and come back to them later:

```Shell
git checkout -b new_branch  # You might want a more descriptive name
git commit -a -m 'Some descriptive message'
git checkout master
git pull
```

Now, you can continue working on this pull request, and whenever you want to go back to those changes, you would `git checkout new_branch`.